apps/ledger-live-mobile/src/screens/Account/index.tsx

```diff
@@ -48,9 +49,20 @@ const AnimatedFlatListWithRefreshControl = Animated.createAnimatedComponent(
   accountSyncRefreshControl(FlatList),
 );
 
+/** If deep linking params are present, this Account Screen is redirected to from Accounts Screen. */
 function AccountScreen({ route }: Props) {
   const { account, parentAccount } = useSelector(accountScreenSelector(route));
+
+  const dispatch = useDispatch();
+
+  /** Reset privacy lock after content has loaded. */
+  useEffect(() => {
+    const timeout = setTimeout(() => dispatch(setIsDeepLinking(false)), 500);
```

Comment:
Not a fan of timeouts but there is no "event" to latch on to - `useEffect` and `useFocusEffect` run too early and lock the screen.

Reply:
Could you do it depending on if data has been loaded?

Reply:
have refactored this into a different method so there's only one timeout now